jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/JKubeConfigurationTest.java

```diff
@@ -89,6 +90,32 @@ void inSourceDir_withRelativePath_shouldReturnResolvedPath() {
     assertThat(result).isEqualTo(new File("/src/other"));
   }
 
+  @Test
+  void builder() {
+    // Given
+    JKubeConfiguration.JKubeConfigurationBuilder builder = JKubeConfiguration.builder()
+        .project(JavaProject.builder().artifactId("test-project").build())
+        .sourceDirectory("src/main/jkube")
+        .outputDirectory("target")
+        .buildArgs(Collections.singletonMap("foo", "bar"))
+        .registryConfig(RegistryConfig.builder()
+            .registry("r.example.com")
+            .build())
+        .resolvedResourceSourceDirs(Collections.singletonList(new File("src/main/jkube")));
```

Comment:
What are "resolved resource source dirs" ? what is the difference with a "source directory"? Maybe this could be clarify with a better naming and example here?